.github/workflows/build-and-test.yml

```
@@ -188,8 +216,8 @@ jobs:
           command: test
           args: --manifest-path ./bindings/wasm/Cargo.toml --release
 
-      - name: Run wasm-bindgen tests # tests annotated with #[wasm_bindgen_test]
-        run: wasm-pack test --node
+      - name: Test Wasm unit and examples
+        run: npm run test
```

Nit: name, optional.

```suggestion
      - name: Run Wasm unit tests and examples
        run: npm run test
```